R/copy-model-from.R

```diff
@@ -4,19 +4,36 @@
 
 #' Create new model by copying existing model
 #'
-#' Create new model by copying existing model. Useful for iterating during model development.
-#' Also fills `based_on` field by default, for constructing model ancestry. See ["Using based_on field" vignette](../articles/using-based-on.html) for details.
+#' Create new model by copying existing model. Useful for iterating during model
+#' development. Also fills `based_on` field by default, for constructing model
+#' ancestry. See ["Using based_on field"
+#' vignette](../articles/using-based-on.html) for details.
 #' @param .parent_mod Model to copy from
-#' @param .new_model Path to write new model files to WITHOUT FILE EXTENSION. Function will create both `{.new_model}.yaml` and a new model file based on this path.
-#' @param .description Description of new model run. This will be stored in the yaml (to be used later in `run_log()`).
-#' @param .based_on_additional Character vector of path(s) to other models that this model was "based on." These are used to reconstuct model developement and ancestry. **Paths must be relative to `.new_model` path.** Note that the `.parent_model` will automatically be added to the `based_on` field, so no need to include that here.
-#' @param .add_tags Character vector with any new tags(s) to be added to `{.new_model}.yaml`
-#' @param .inherit_tags If `FALSE`, the default, new model will only have any tags passed to `.add_tags` argument. If `TRUE` inherit any tags from `.parent_mod`, with any tags passed to `.add_tags` appended.
-#' @param .update_model_file If `TRUE`, the default, update the newly created model file with new description and name.
-#' For a NONMEM model, this currently means only the `$PROBLEM` line in the new control stream will be updated.
-#' If `FALSE`, new model file will be an exact copy of its parent.
-#' @param .overwrite If `FALSE`, the default,  function will error if a model file already exists at specified `.new_model` path. If `TRUE` any existing file at `.new_model` will be overwritten silently.
-#' @param .directory Model directory which `.new_model` is relative to. Defaults to `options('rbabylon.model_directory')`, which can be set globally with `set_model_directory()`.
+#' @param .new_model Path to write new model files to **without file extension**.
```

Comment:
I'm a little annoyed we did this reformat. We kept the diff so clean otherwise.

Reply:
On the plus side, it's easier to read (no horizontal scrolling).